mqtt.go

```diff
@@ -147,6 +147,8 @@ func (client *MQTTClient) Run() {
 		}
 	}
 
+	client.Publish(fmt.Sprintf("parasite-scanner/status"), fmt.Sprintf("online"), false, 1)//publish online status
```

Comment:
I think we can drop the `fmt.Sprintf` and just use the literal string `"online"` instead.

Would you also please run the code through `go fmt`? Tks!

Reply:
You are right, ommiting the fmt stuff works. I have zero experience with go and have just copy/pasted a bit. 

`go fmt` just added a space before the comment. That comment is pretty useless anyways so i just deleted it.